ai_feedback/__main__.py

```diff
@@ -148,6 +150,13 @@ def main() -> int:
         required=False,
         help=HELP_MESSAGES["solution_image"]
     )
+    parser.add_argument(
+        "--system_prompt",
+        type=str,
+        required=False,
+        help=HELP_MESSAGES["system_prompt"],
+        default="ai_feedback/data/prompts/system/student_test_feedback.md"
```

Comment:
I'd like to replicate the same behaviour as `--prompt`, where any built-in prompts can be specified by name only (e.g. `student_test_feedback`), but if another name is specified then it's treated as a filepath (to a user-provided custom system prompt).

Reply:
Whoops, I forgot that currently we don't allow a filepath to a custom user prompt. That's fine, we can add it later. In the mean time, make the behaviour the same as the existing user prompts.